app/src/main/res/values/strings.xml

```diff
@@ -65,7 +65,7 @@
   <string name="refresh_button">Refresh</string>
   <string name="display_list_button">List</string>
   <string name="gps_disabled">GPS is disabled in your device. Would you like to enable it?</string>
-  <string name="enable_gps">Enable GPS</string>
+  <string name="enable_gps">Enable Gps</string>
```

Comment:
GPS is stands for Global Positioning System. Please do not change abbreviations like GPS, FAB etc. Just make sure there are no normal word which is written in all caps.

Reply:
And I didn't understand why you changed just one GPS while other GPS is still all caps, I would expect all GPSs are written in the same way. Consistency is important. In our case please leave all GPSs as all capital though.

Reply:
ok i change is done